web/cobrands/sass/_layout.scss

```diff
@@ -821,33 +833,38 @@ textarea.form-error {
 
 // big yellow bit full screen width
 #front-main {
-  color: $primary_text;
-  background: $primary;
-  margin: 0;
+  color: $front-main-color-desktop;
+  background: $front-main-background-desktop;
   padding: 1em;
+  margin: 0 auto;
   #front-main-container {
     max-width: $container-max-width;
     margin: 0 auto;
+    background: $front-main-container-background-desktop;
+    @if not (($front-main-container-background-desktop == transparent) or ($front-main-container-background-desktop == none)) {
+      padding: 1em;
+    }
   }
   h2 {
     font-style:normal;
     margin:0;
-    color: inherit;
-  }
-  #postcodeForm {
-    color: inherit;
-    background:none;
-    overflow:hidden;
-    padding-bottom: 0;
```

Comment:
Why was this dropped? e.g. Bexley has a lot more bottom padding now than it used to.